.github/README.md

```diff
@@ -0,0 +1,166 @@
+## What are GitHub Actions?
+
+GitHub Actions are used for many different purposes.  We use them to run tests in CI, validate PRs are in an expected state, and automate processes.
+
+- [Overview of GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions)
+- [What's a workflow?](https://docs.github.com/en/actions/using-workflows/about-workflows)
+- [GitHub Actions guides](https://docs.github.com/en/actions/guides)
+
+## dbt Labs Action Repo
+[dbt-labs/actions](https://github.com/dbt-labs/actions/) is a central repository of actions and workflows dbt uses across repositories.
+
+## General Standards
+
+### Permissions
+- Default to least permissions.
+- There are a lot of permissions available.  [Read up on them](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs) if you're unsure what to use.
+
+### using on_merge?
+- TODO: what was the security hole related to this?
+    
+### Secrets
+- When to use a [Personal Access Token (PAT)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) vs the [GITHUB_TOKEN](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) generated for the action?
+
+    If you expect the resulting commit to add a changelog should retrigger workflows, you will need to use a Personal Access Token for the bot to commit the file. When using the GITHUB_TOKEN, the resulting commit will not trigger another GitHub Actions Workflow run. This is due to limitations set by GitHub. See [the docs](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow) for a more detailed explanation.
```

Comment:
I would state that you are using the `GITHUB_TOKEN` by default and that in most cases that is all you need. For chaining workflow runs together (i.e. workflow A triggers workflow B), those are special cases where GitHub doesn't want to cause an infinite loop of workflows triggering and that is when you need to explicitly use a PAT